app/controllers/address_controller.rb

```diff
@@ -0,0 +1,35 @@
+class AddressController < ApplicationController
+  def create
+    user = User.find_by(verification_token: params[:verification_token])
+    address = user.addresses.build(permitted_address_params)
+
+    if address.save
+      order = logged_in_user_cart_order
```

Comment:
let's rename `logged_in_user_cart_order` to `current_order`

Reply:
Renamed